### PR TITLE
Improve infrastructure colors and logos

### LINKED ISSUE #18: Improve Infrastructure Labels, Colors and Logos
Companies may want to "brand" their supported infrastructures, e.g. internal OpenStack installations. Historically the Dashboard used SAP Gold for "SAP Cloud Platform" (now "OpenStack"), but we should change this. Also the other infrastructures use not the right logos/colors, so it's time for a cleanup.

Let's make the following bits configurable at Garden cluster resource level (not hard-coded):
* Name, RGB color, (abstract) resource name (for logo) per cloud profile

If we have this, it should be possible to support also multiple AWSes (e.g. general and government cloud), multiple OpenStacks or whatnot. It will also be possible to "brand" them, e.g. "OpenStack" could become "OpenStack (SAP Cloud Platform on Converged Cloud Industry Edition)" or we could integrate "special" discriminators/qualifiers like "Office Network" into the configurable name.

Let's also change the following:
* OpenStack color should rather become OpenStack Red (see logo below)
* Logo -> https://www.openstack.org/brand/openstack-logo/logo-download

image

* AWS: https://aws.amazon.com/de/architecture/icons -> AWS Simple Icons im EPS- und SVG-Format -> Simple Icons im EPS- und SVG-Format or AWS Simple Icons für PowerPoint

screen shot 2018-07-24 at 09 19 44

* Azure: New logo (not sure where to get it from officially), see: https://buildazure.com/2017/09/25/microsoft-azure-gets-a-new-logo-and-a-manifesto/

image

* GCP: https://cloud.google.com/icons -> SVG/PNG -> Products & Services/Extras/Google Cloud Platform.png|svg

0eaf91b4-e08c-11e7-9ef2-67a651644da4

### LINKED ISSUE #168: miss aligned icon in delete secret dialog
## bad aligned icon
bildschirmfoto 2018-08-14 um 14 30 26

## good aligned icons

bildschirmfoto 2018-08-14 um 14 30 50

bildschirmfoto 2018-08-14 um 14 30 58

### LINKED ISSUE #366: miss aligned icons in the cluster overview
Icon for aliCloud is not on baseline.

Bildschirmfoto 2019-04-17 um 13 40 56

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
We are now using the correct infrastructure icons. Also the colors were aligned.

Screenshot 2019-04-29 at 14 53 39

Screenshot 2019-04-29 at 14 52 57

**Which issue(s) this PR fixes**:
Fixes #18 Fixes #366 

**Special notes for your reviewer**:

**Release note**:

```improvement user
Improved infrastructure colors and logos
```

### CODE REVIEW

**Comment by vlerenc** [comment]
Thank you @petersutter !

Cosmetic comment on the release note: "icons and logos" or did you mean "colors and logos" like in the PR title?

**Comment by petersutter** [comment]
Yes, thanks for letting me know! I have corrected the release note.

File: frontend/src/App.vue [hunk 1]
```diff
@@ -43,5 +43,18 @@ export default {
     .container {
       padding: 16px;
     }
+
```
Comment (petersutter) [inline frontend/src/App.vue]: move to stylus/main and define color values also there

File: frontend/src/components/ShootListRow.vue [hunk 2]
```diff
@@ -41,11 +41,10 @@ limitations under the License.
     </td>
     <td class="nowrap" v-if="this.headerVisible['infrastructure']">
       <v-tooltip top>
-        <div slot="activator">
-          <img v-if="row.kind === 'alicloud'" src="@/assets/alicloud.svg" width="20" class="mr-2">
-          <infra-icon v-else v-model="row.kind"></infra-icon>
+        <v-layout align-center justify-start row fill-height slot="activator">
```
Comment (petersutter) [inline frontend/src/components/ShootListRow.vue]: check if layout is really needed

File: frontend/src/components/ShootListRow.vue [hunk 3]
```diff
@@ -41,11 +41,10 @@ limitations under the License.
...
+          <infra-icon v-model="row.kind" content-class="mr-2"></infra-icon>
           {{ row.region }}
```
Comment (petersutter) [inline frontend/src/components/ShootListRow.vue]: surround with div

File: frontend/src/pages/Secrets.vue [hunk 4]
```diff
@@ -154,19 +154,19 @@ limitations under the License.
           <v-icon>close</v-icon>
         </v-btn>
         <v-btn v-if="hasCloudProfileForCloudProviderKind('alicloud')" fab dark small class="grey darken-4" @click="onAdd('alicloud')">
-          <img src="@/assets/alicloud-white.svg" width="20">
+          <infra-icon value="alicloud-white" :width="20"></infra-icon>
         </v-btn>
-        <v-btn v-if="hasCloudProfileForCloudProviderKind('openstack')" fab dark small class="orange" @click="onAdd('openstack')">
-          <v-icon>mdi-server-network</v-icon>
+        <v-btn v-if="hasCloudProfileForCloudProviderKind('openstack')" fab dark small color="#ED1944" @click="onAdd('openstack')">
```
Comment (petersutter) [inline frontend/src/pages/Secrets.vue]: create utility function for infra color

**Pull request merged**